tools/istio-iptables/pkg/capture/run_test.go

```diff
@@ -356,6 +357,9 @@ func TestSeparateV4V6(t *testing.T) {
 }
 
 func TestIdempotentEquivalentRerun(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		t.Skip("Skipping test: Not running on Linux")
```

Comment:
I opted not to skip building the whole test file, as run_test.go also contains non-e2e tests that use DependenciesStub

Reply:
maybe you can split them into another test file with `go:build` tag.

Reply:
Done